api_server.go

```diff
@@ -32,8 +33,9 @@ func NewApiServer(store ObjectStore, config ApiServerConfig) *ApiServer {
 		ApiServerConfig: config,
 		ServeMux:        http.NewServeMux(),
 	}
-	server.Handle("/sheets", APIHandlerFunc(server.MusicPDFsIndex))
-	server.Handle("/sheets/upload", APIHandlerFunc(server.MusicPDFsUpload))
+	server.Handle("/sheets", APIHandlerFunc(server.SheetsIndex))
+	server.Handle("/sheets/upload", APIHandlerFunc(server.SheetsUpload))
+	server.Handle("/download", http.HandlerFunc(server.Download))
```

Comment:
For consistency, this should be `""/sheets/download"`, and `server.SheetsDownload`.

Reply:
This api works for any object download.

Reply:
I guess we don't really need to restrict the endpoint to sheets for now.